packages/elements/src/photon-multirx-form/photon-prescribe-workflow-component.tsx

```diff
@@ -1,9 +1,20 @@
 import { RecentOrders } from '@photonhealth/components';
 import { customElement } from 'solid-element';
 import { createFormStore } from '../stores/form';
+import { PatientStore } from '../stores/patient';
 import { PrescribeProps, PrescribeWorkflow } from './photon-prescribe-workflow';
+import { createEffect, onMount } from 'solid-js';
+import { usePhoton } from '../context';
 
 const Component = (props: PrescribeProps) => {
+  const client = usePhoton();
+  const { actions: patientActions, store: patientStore } = PatientStore;
+  onMount(() => {
+    if (props.patientId) {
+      patientActions.getSelectedPatient(client!.getSDK(), props.patientId);
```

Comment:
is there not logic somewhere that's already doing this? we're presumably already doing this since the patient card already loads if you pass in a patient id